part3.md

```diff
@@ -0,0 +1,80 @@
+## Part 3 - Hello Kotlin
+
+ * Make sure that you have Kotlin JS installed (in `functions` sub-folder)
+```
+npm install -g kotlin
+```
+
+ * Open IntelliJ IDEA and create new pure Kotlin project in root folder. Use Kotlin(JavaScript) and Gradle in wizard. As a result you shold have `functions` folder as a subfolder of you project. 
```

Comment:
a new
optionally ~pure~
the root
the root folder (of what?)

In the setup wizard select `Kotlin(JavaScript)` as the [what the screen title says] and `Gradle` as the/your [what].
As a result the folder `functions` should now [effectively] be a sub-folder [or is it subfolder? consistency!] of your [Kotlin [JS]] project.